### ISSUE TITLE
buttons unresponsive

### ISSUE THREAD

**Issue by CodingArctic** [d093w1z/polybar-now-playing#1]
glad I found this project because I was looking for this exact thing, but now after setting it up, the buttons don't respond to clicks. any ideas here? I've tried running the script non-daemonized to see if theres any error messages but there don't seem to be.

**Comment by d093w1z** [comment]
The buttons are made responsive using [Action Formatting](https://github.com/polybar/polybar/wiki/Formatting#action-a).
Providing configuration would be helpful.
Do other modules(custom/default) respond to clicking?
Does another test script respond to click events using Action formatting ?

Thanks for waiting.

**Comment by CodingArctic** [comment]
this is what i have in my modules.ini file:
`[module/now-playing]
type = custom/script
tail = true
;format-prefix = ""
format = <label>
exec = /home/arctic/.config/polybar/polybar-now-playing
click-right = "kill -URL1 $(pgrep --oldest --parent %pid%)"
`

I didn't create the polybar originally but other modules are responsive to clicks. (I'm not sure how I would create a test script.)

**Comment by d093w1z** [comment]
There's no problem with the config.
>  I didn't create the polybar originally ......

Could you rephrase this in more expressive terms?

Also, what is the output after killing polybar and running  `polybar -l info <name-of-bar>` ?
And does the script(python file) on itself print the desired output after running it manually(running `python3 polybar-now-playing` or `./polybar-now-playing`)?
For clarification, desired output here means a string containing the action tags(something like `"formatted text %{A: playerctl command:} control-button %{A}"`).

**Comment by CodingArctic** [comment]
this is the output of that: 
![image](https://user-images.githubusercontent.com/82346273/120083885-e5087c80-c099-11eb-860d-382ce1dbe1be.png)


for some reason now it doesn't seem to be hooking into any apps. just shows this:
![image](https://user-images.githubusercontent.com/82346273/120085346-703b3f80-c0a5-11eb-97c5-0069d0906b83.png)

**Comment by d093w1z** [comment]
>  this is the output of that:

You also need to specify the path to configuration file as suggested in the error.

> for some reason now it doesn't seem to be hooking into any apps. just shows this:

This one doesn't mean that it's not able to hook into any application, if it shows No title or No artist means that the current playing  media does not have a title or artist.

>edit: after rebooting it started showing up again. this is console output of the script, while repeatedly pressing the play/pause button (seemingly no response):

Based on the output, the script runs fine. This doesn't seem to be a problem of the python script.

I would definitely help and make changes if there's a bug in my code.
But if the problem is with your specific setup, i would advice you to read up on the configuration pages and look for a problem.

**Comment by CodingArctic** [comment]
@d093w1z really sorry for the inconvenience. after looking at your script i realized that the buttons try to run playerctl commands, and i thought i had that package but i hadn't, lol. anyways, it works now, thank you.
